### PR TITLE
Add Felipe & Ben's examples using binary diversification tools

### PR DESCRIPTION
**Pull request by codyroux**

### CODE REVIEW

File: wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+dummy_dir=../../dummy
+
+compile () {
+  make
+}
+
+run () {
+  bap $dummy_dir/hello_world.out --pass=wp \
+    --wp-compare=true \
+    --wp-inline \
```
Comment (fortunac) [inline wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh]: We changed how the inline flag works. To inline everything, we should `--wp-inline=.*`

Comment (codyroux) [reply wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh]: Oh right. The example crashes regardless, I believe.

Comment (codyroux) [reply wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh]: But I'll fix it.

Comment (fortunac) [reply wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh]: Sounds good!

Comment (codyroux) [reply wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh]: All right, fixed.

**Pull request merged**